unity-renderer/Assets/Scripts/MainScripts/DCL/Components/AvatarAttach/AvatarAttachHandler.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace DCL.Components
 {
-    internal class AvatarAttachHandler : IDisposable
+    public class AvatarAttachHandler : IDisposable
```

Comment:
could we move this file to the ecs7 component folder since the previous component will be deprecated?